starcounter-include.html

```diff
@@ -288,6 +289,60 @@
             return range.createContextualFragment(htmlStr);
         }
 
+        /**
+         * Replaces all <slot> tags with <content>,
+         *  preserves all attributes, translates selector to slot name
+         * @param  {String} string stringified HTML
+         * @return {String}        translated HTML
+         */
+        function replaceSlotsWithContentInString(string){
+            return string.replace(/<\/slot(\s*)>/gi,'</content$1>')
+                    .replace(/<slot([^>]+)(name=(?:"([^"]*)"|'([^']*)'))([^>]*)>/gi,'<content$1$2$5 select=\"[slot=\'$3$4\']\">')
+                    .replace(/<slot(\s*|\s+[^>]*)>/gi,'<content$1>');
+        }
+        /**
+         * Replaces single SlotElement with ContentElement,
+         *  preserves all attributes, translates selector to slot name
+         * @param  {SlotElement} slot
+         * @return {ContentElement}
+         */
+        function replaceSlotWithContent(slot) {
+            var content = slot.ownerDocument.createElement('content');
+            while (slot.firstChild) {
+                content.appendChild(slot.firstChild);
+            }
+            var attrs = slot.attributes;
+            for (var i=0; i<attrs.length; i++) {
+                var attr = attrs[i];
+                content.setAttribute(attr.name, attr.value);
+            }
+            var name = slot.getAttribute('name');
+            if (name) {
+                content.setAttribute('select', '[slot=\'' + name + '\']');
+            }
+            slot.parentNode.replaceChild(content, slot);
+            return content;
+        }
+        /**
+         * Replace all <slot> elements with <content> elements i given document fragment
+         * @param  {Documentfragment|Node} root scope. Will be changed.
+         * @return {Documentfragment|Node}      modified scope
+         */
+        function replaceSlotsWithContent(root) {
+            if (root.firstChild) {
+                var node = root.firstChild;
+                while (node) {
+                    var next = node.nextSibling;
```

Comment:
It's for weird behaving custom elements in fallback content, that might mess around, plus I'm not sure how Shadow DOM dom polyfill distributes elements. Polymer does so in their shim.

Have you tested all the browsers?